docs/FAQ.md

```diff
@@ -0,0 +1,26 @@
+# Frequently asked questions
+
+### Can we use Adyen web components >= v5.0.0 with this integration ?
+
+Yes, you could use the web components > v5.0.0. It's always better to keep your web component front end dependencies up to date with following latest Adyen releases.
+
+In v5.0.0 (released in October 2021) Adyen introduced a simplified way of integrating Web Components, using a single API endpoint. The /sessions release is only an orchestration layer on top of existing functionalities.
+This means that the complete checkout experience is still available and existing merchants will not have to change anything to their existing workflow. In fact, the 3 step /paymentMethods, /payments, /payments/details is still the way to go for more complex user flows.
+Adyen will be revisiting this topic in Q2 2022 and we will keep you updated on the potential upgrade regarding with /session endpoint usage with this integration.
```

Comment:
You can not use with integration now maybe in future if we add support for it with integration. as described this is not necessary for now until Adyen inform us to do a change.

Reply:
But here you have this:

### Can we use Adyen web components >= v5.0.0 with this integration ?

Yes, you could use the web components > v5.0.0.

Reply:
I meant for session endpoint, but you could use the web components (js) and you should keep the deps up to date, wdyt how should we document this one. 
See [this](https://docs.adyen.com/online-payments/web-components/integrated-before-5-0-0) page it's mentioned as: "even as you update your Components version."

Reply:
Hmm for me the update to new JS frontend version did not work, because I did not have the session thing (see our slack conversation with Adyen). That's why I think this statement is not correct. For now I would say `No, it's not possible to use web components >= 5.0.0. Please follow the documentation before v5.0.0: https://docs.adyen.com/online-payments/web-components/integrated-before-5-0-0`. You can leave the rest of your answer as it is.

Maybe you can also clarify this statement in the documentation with Adyen? Seems like it's not true.

Reply:
Removed the statement for now, will check later based on the discussion. c6d1ce4